### ISSUE TITLE
Add a link to reset password on login page

### ISSUE THREAD

**Issue by FafnerKeyZee** [MISP/MISP#4779]
Hey,

It could be nice to add the possibility to add (or not) a link to the main page to reset a forgotten password.
Depending the context of the instance, it would be great to manage this possibility using the configuration. 

Best regards,

**Comment by chicagocyb3r** [comment]
I agree that the option for a "reset password" functionality would be an excellent addition.
